applications/crossbar/priv/api/swagger.json

```diff
@@ -12702,37 +12702,9 @@
         "version": "2.0"
     },
     "paths": {
-        "/v2/about": {
-            "get": {}
-        },
```

Comment:
I wanted to avoid magic ways of finding relevant information. I don't want developers to have to remember to add an attribute to tag the module as root, masqueraded, both, etc. I think encoding the "root" modules in cb_api_endpoints makes the most sense. When a new "root" endpoint is developed, the ref doc will have `/v2/accounts/{ACCOUNT_ID}/new_endpoints` and such. The dev creating the new endpoint will see that and have to adjust cb_api_endpoints to account for the new endpoint. I don't see this being a frequent-enough occurrence to warrant compiler attributes.

We can discuss more after the standup.